modules/fedimint-wallet-tests/tests/tests.rs

```diff
@@ -1053,3 +1053,32 @@ mod fedimint_migration_tests {
         .await
     }
 }
+
+// Verify the correct Bitcoin RPC is used
+
+#[cfg(feature = "bitcoind")]
```

Comment:
Another approach is using `#[ignore]`, but that makes the calls to `cargo nextest` messier since it requires multiple flags and filters.